frontend/server/src/DAO/Submissions.php

```diff
@@ -313,4 +313,26 @@ public static function getLatestSubmissions(
             'submissions' => $submissions,
         ];
     }
+
+    /** @return \OmegaUp\Timestamp|null */
+    public static function getLastSubmissionByProblem(
```

Comment:
```suggestion
    public static function getLastSubmissionTimestampByProblem(
```

esto regresa el timestamp del submission, no el submission en sí.